easybuild/easyconfigs/p/PyZMQ/PyZMQ-22.1.0-foss-2020b.eb

```
@@ -0,0 +1,33 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonPackage'
+
+name = 'PyZMQ'
+version = '22.1.0'
+local_zmqversion = '4.3.3'
+
+homepage = 'https://www.zeromq.org/bindings:python'
+description = """Python bindings for ZeroMQ"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7040d6dd85ea65703904d023d7f57fab793d7ffee9ba9e14f3b897f34ff2415d']
+
```

```suggestion

builddependencies = [('binutils', '2.35')]

```
`GCCcore` toolchain will require `binutils` as a build dep.